### PR TITLE
All claims extra ptsd triggers mg

### PR DESCRIPTION
**Pull request by markgreenburg**
## Description
- Adds some more 'trigger words' to indicate that a newly-entered disability is actually PTSD
- Updates the checker to include both instances where one of our trigger strings is in the disability name that was typed, *or* that the disability name that was types is in one of the trigger strings
- Updates references to old PTSD constant to instead make use of the `isPtsd` helper function
- Removes some unnecessary lodash usage

## Testing done
- Tested locally (that the PTSD page triggers when it should and vice-versa for non-PTSD follow-up pages).

## Screenshots
- NA

## Acceptance criteria
- [x] Allow other PTSD-ish things to trigger 781/a follow up pages

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by markgreenburg** [comment]
@djmunoz1 let me know if there are any other triggers you want:
  'ptsd',
  'post traumatic stress disorder',
  'post-traumatic stress disorder',
  'post traumatic stress',
  'post-traumatic stress',

We'll consider a new name a match if either:
- The name they typed in is fully contained in one of the names above
- One of the names above is fully contained within a disability they type in
- The disability name they type in exactly matches one of the names above

File: src/applications/disability-benefits/all-claims/utils.jsx [hunk 1]
```diff
@@ -381,13 +381,23 @@ const post911Periods = createSelector(
...
+      ptsdString.includes(loweredDisability) ||
+      loweredDisability.includes(ptsdString),
+  );
+};
 
 export const hasNewPtsdDisability = formData =>
   _.get('view:newDisabilities', formData, false) &&
-  some(_.get('newDisabilities', formData, []), item =>
-    isDisabilityPtsd(item.condition),
+  _.get('newDisabilities', formData, []).some(disability =>
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/utils.jsx]: This didn't strictly need to be in this PR but I really wanted to cut out the unnecessary lodash usage while I was already cutting out lodash's `lowerCase()`.

File: src/applications/disability-benefits/all-claims/utils.jsx [hunk 2]
```diff
@@ -381,13 +381,23 @@ const post911Periods = createSelector(
...
+export const isDisabilityPtsd = disability => {
+  if (!disability || typeof disability !== 'string') {
+    return false;
+  }
+
+  const loweredDisability = disability.toLowerCase();
+  return PTSD_MATCHES.some(
+    ptsdString =>
+      ptsdString.includes(loweredDisability) ||
+      loweredDisability.includes(ptsdString),
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/utils.jsx]: I made [a little JS Bin](https://jsbin.com/jobiciyoni/edit?js,console) to test out a theory. Not sure if you want to use it, but I think we can make this use regular expressions as well to accomplish the same task.

File: src/applications/disability-benefits/all-claims/utils.jsx (hunk 1 above)
Comment (cvalarida) [reply src/applications/disability-benefits/all-claims/utils.jsx]: I also dig the argument rename. :100:

**Review by cvalarida — APPROVED** [review]
Looks great!

Might write a couple tests for `isDisabilityPtsd` to catch strings it should and shouldn't match.

File: src/applications/disability-benefits/all-claims/utils.jsx (hunk 2 above)
Comment (markgreenburg) [reply src/applications/disability-benefits/all-claims/utils.jsx]: This is nice; when I wrote this thing I had a suspicion there was a way to do this with regex 😄 
After thinking about it for a bit, I think I'm going to stick with the current approach, for no other reason than the fact that it's subjectively easier for me to keep track of what's going on 😛 .

Comment (cvalarida) [reply src/applications/disability-benefits/all-claims/utils.jsx]: Hehe, 's fine by me. :stuck_out_tongue:

**Pull request merged**